digdag-tests/src/test/java/acceptance/CliShowProjectsIT.java

```diff
@@ -0,0 +1,169 @@
+                "-c", config.toString(),
+                projectDir.toString());
+        assertThat(initStatus.code(), is(0));
+
+        copyResource("acceptance/basic.dig", projectDir.resolve("basic.dig"));
+
+        // digdag push first project. (named: foo)
+        {
+            CommandStatus pushStatus = main(
+                    "push",
+                    "--project", projectDir.toString(),
+                    "foo",
+                    "-c", config.toString(),
+                    "-e", server.endpoint());
+            assertThat(pushStatus.errUtf8(), pushStatus.code(), is(0));
+        }
+
+        // digdag push second project. (named: bar)
+        {
+            CommandStatus pushStatus = main(
+                    "push",
+                    "--project", projectDir.toString(),
+                    "bar",
+                    "-c", config.toString(),
+                    "-e", server.endpoint());
+            assertThat(pushStatus.errUtf8(), pushStatus.code(), is(0));
+        }
+
+        CommandStatus showProjectStatus = main("projects","foo",
+                "-c", config.toString(),
+                "-e", server.endpoint());
+
+        assertThat(showProjectStatus.errUtf8(), showProjectStatus.code(), is(0));
+        assertThat(showProjectStatus.outUtf8(), containsString(" name: foo"));
+        // This assert expect `digdag projects foo` doesn't show project `bar` detail.
+        assertThat(showProjectStatus.outUtf8(), not(containsString(" name: bar")));
+    }
+    @Test
+    public void showProjectWithNonRegisteredProject()
+            throws Exception
+    {
+
+        // Create new project
+        CommandStatus initStatus = main("init",
+                "-c", config.toString(),
+                projectDir.toString());
+        assertThat(initStatus.code(), is(0));
+
+        copyResource("acceptance/basic.dig", projectDir.resolve("basic.dig"));
+
+        // digdag push first project. (named: foo)
+        {
+            CommandStatus pushStatus = main(
+                    "push",
+                    "--project", projectDir.toString(),
+                    "foo",
+                    "-c", config.toString(),
+                    "-e", server.endpoint());
+            assertThat(pushStatus.errUtf8(), pushStatus.code(), is(0));
+        }
+
+        // digdag push second project. (named: bar)
+        {
+            CommandStatus pushStatus = main(
+                    "push",
+                    "--project", projectDir.toString(),
+                    "bar",
+                    "-c", config.toString(),
+                    "-e", server.endpoint());
+            assertThat(pushStatus.errUtf8(), pushStatus.code(), is(0));
+        }
+
+        CommandStatus showProjectStatus = main("projects","not_exist_project",
+                "-c", config.toString(),
+                "-e", server.endpoint());
+
+        assertThat(showProjectStatus.errUtf8(), showProjectStatus.code(), is(0));
```

Comment:
Is exit code 0 is right when resource not found?
It looks like other commands such as 'workflows' return non-zero if resource not found. 

```
$ digdag workflows not_exist_workflow || echo fail
2019-04-10 14:35:53 +0900: Digdag v0.9.35
error: Resource not found: project not found: not_exist_workflow
fail
```